src/main/java/tech/fastj/graphics/game/package-info.java

```diff
@@ -0,0 +1,5 @@
+package tech.fastj.graphics.game;
+
+/**
+ * Different elements meant to be used in a FastJ game.
```

Comment:
This is a bit too broad of a topic -- "different elements" what elements? We can't tell anything about these elements other than that they should be used in a FastJ game, and _quite literally anything_ in FastJ's source could or maybe even should be used in a FastJ game.

Consider a more focused approach: "{\@code GameObject}s and specific instances".